FlowCrypt/Controllers/Inbox/InboxViewController.swift

```diff
@@ -81,31 +85,27 @@ class InboxViewController: ViewController {
     override func viewDidLayoutSubviews() {
         super.viewDidLayoutSubviews()
 
-        tableNode.frame = node.bounds
-
-        if isSearch { return }
+        guard !didLayoutSubviews else { return }
 
-        let offset: CGFloat = 16
-        let size = CGSize(width: 50, height: 50)
+        setupElements()
+        ekmVcHelper.refreshKeysFromEKMIfNeeded(in: self)
```

Comment:
I wonder why this logic moved to InboxViewController.
And I think with this change, refresh logic will run every time I switch between inbox folders.

Reply:
ok